### PR TITLE
[5] Configuring CI to run against several versions of Rails and Ruby

### LINKED ISSUE #5: Configure CI to run against several rails versions
## Why
We want to test the gem against several rails versions so that we are aware the gem is working correctly for everyone

## What
Update our Travis CI configuration to use Rails 4, 5 and 6 as the Rails target. And keep using Ruby 2.5, 2.6 and 2.7.

## Notes
Github Actions is [supposedly faster than Travis](https://twitter.com/nateberkopec/status/1238945862476271617). Check if it's easier to configure Github Action instead of Travis CI

### PR DESCRIPTION
**Pull request by juanmanuelramallo**
Resolves #5

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Tests and Code Analysis
```
Comment (santib) [inline .github/workflows/ci.yml]: I think we now should remove `.travis.yml`, and uninstall travis from the checks

Comment (santib) [reply .github/workflows/ci.yml]: but at the same time we need the code analysis task to continue running 🤔

File: lib/yaaf/form.rb [hunk 2]
```diff
@@ -41,10 +41,9 @@ def save_in_transaction(options)
         run_callbacks :save do
           save_models(options)
         end
-
-      rescue Exception => e
-        handle_transaction_rollback(e)
       end
+    rescue Exception => e
```
Comment (santib) [inline lib/yaaf/form.rb]: why is this change?

Comment (juanmanuelramallo) [reply lib/yaaf/form.rb]: https://github.com/rootstrap/yaaf/pull/20/checks?check_run_id=543708276

**Pull request merged**